home/HomeApp.tsx

```diff
@@ -114,6 +115,12 @@ export default function HomeApp() {
   );
 }
 
+// Certain links (i.e. 'expo.io/expo-go') should just open the HomeScreen
+function shouldOpenUrl(urlString: string) {
+  const url = new URL(urlString);
```

Comment:
do we define URL? make sure to test this live

Reply:
changed to `url.parse`, tested and works as expected